README.md

```diff
@@ -1,12 +1,72 @@
+[![License](https://img.shields.io/github/license/CybercentreCanada/assemblyline-service-server)](./LICENSE.md)
+
 # Assemblyline 4 - Service Server
 
 The service server is a API that the service clients can call to interface with the system. This is the only access the services have to the system as they are completely segregated from the other components.
 
-##### API functionality
+## Image variants and tags
+
+| **Tag Type** | **Description**                                                                                  |      **Example Tag**       |
+| :----------: | :----------------------------------------------------------------------------------------------- | :------------------------: |
+|    latest    | The most recent build (can be unstable).                                                         |          `latest`          |
+|  build_type  | The type of build used. `dev` is the latest unstable build. `stable` is the latest stable build. |     `stable` or `dev`      |
+|    series    | Complete build details, including version and build type: `version.buildType`.                   | `4.5.stable`, `4.5.1.dev3` |
+
+## API functionality
 
 Service server provides the following functions via API to the client:
 
-* File download and upload
-* Register service to the system
-* Get a new task
-* Publish results for a task
\ No newline at end of file
+- File download and upload
+- Register service to the system
+- Get a new task
+- Publish results for a task
+- Checking if certain tags or files have a reputation relative to the system (ie. safe vs malicious)
+
+#### Running this component
+
+```bash
+docker run --name service-server cccs/assemblyline-service-server
+```
+
+## Documentation
+
+For more information about this Assemblyline component, follow this [overview](https://cybercentrecanada.github.io/assemblyline4_docs/overview/architecture/) of the system's architecture.
+
+---
+
+# Assemblyline 4 - Serveur de service
+
+Le serveur de services est une API que les clients des services peuvent appeler pour s'interfacer avec le système. C'est le seul accès des services au système, car ils sont complètement séparés des autres composants.
+
+## Variantes d'images et balises
+
+| **Type d'étiquette** | **Description**                                                                                                                    |  **Exemple d'étiquette**   |
+| :------------------: | :--------------------------------------------------------------------------------------------------------------------------------- | :------------------------: |
+|       dernière       | La version la plus récente (peut être instable).                                                                                   |          `latest`          |
```

Comment:
Should be either just in french or english not both. I feel like it should be only in french in this case so it would be "type de production" et "séries".